### DISCUSSION TITLE
Thoughts on the name

### DISCUSSION THREAD

**Discussion by ChildishGiant** [elementary/switchboard-plug-wacom#27] [General]
I've always found it odd that this plug is called Wacom, not something more generic like "Drawing Tablets". Maybe this is another case of the US using genericized trademarks, such as calling all tissues Kleenex which over here in the UK isn't done. I've personally never heard anyone call a drawing tablet a Wacom, just people referring to the brand.

**Comment by davidmhewitt** [comment]
Yeah, it's an interesting point.

I think it's because it uses libwacom to detect the model of tablet, which in the past definitely used to only support wacom tablets, hence the name.

I think over time, it's picked up support for other brands of tablet, but it still only _officially_ supports Wacom ones. Plus... hardly anyone on the team has drawing tablets, so I think the plug has only been tested with Wacom tablets too.

Not saying I'd be opposed to renaming it, but it does open it up to a lot more issue reports like "My brand X tablet isn't working", which we can do nothing about since we don't have the hardware to test.

**Reply by ChildishGiant** [reply]
Yeah, I do see the potential for people being annoyed if support is only partially there although I imagine if I had a brand x tablet currently I'd still look to the Wacom settings. Maybe changing the wording (eg "No tablets available" -> "No compatible tablets available", probably with a tooltip) would help reduce confusion either way
